projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/cutsomer-ticketing.ts

```diff
@@ -0,0 +1,113 @@
+import { waitForPage, visitHomePage } from '../checkout-flow';
+
+export enum TestCategory {
+  enquiry = "Enquiry",
+  complaint = "Complaint",
+  problem = "Problem",
+}
+
+export interface TestTicketDetails {
+  subject: string;
+  message: string;
+  category: TestCategory;
+  associatedTo?: string;
+  filename?: string;
+}
+
+
+export function visitElectronicTicketListingPage() {
+  visitHomePage();
+  const supportPage = waitForPage('/my-account/support-tickets', 'support-tickets');
+  cy.visit('/my-account/support-tickets');
+  cy.wait(`@${supportPage}`).its('response.statusCode').should('eq', 200);
+}
+
+
+export function openCreateTicketPopup() {
+  cy.get('cx-customer-ticketing-list').should('exist');
+  cy.get('cx-customer-ticketing-create button').click();
+  cy.get('cx-customer-ticketing-create-dialog').should('exist');
+}
+
+export function fillTicketDetails(ticketDetails: TestTicketDetails){
+  const CATEGORY_SELECT = 0;
+
+  // subject
```

Comment:
Why do we need comments.  Please remove of change by local function

export function fillTicketDetails(ticketDetails: TestTicketDetails){
 ...
 function setSubject() {
   cy.get('textarea').first().type(ticketDetails.subject);
}
...
setSubject();
setCategory();
setAssociatedToObject();
...

}